DTDL/ssm_stream_modbus.json

```diff
@@ -0,0 +1,72 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:stream:modbus;1",
+  "@type": "Interface",
+  "extends": "dtmi:io:tributech:stream:base;1",
+  "displayName": "Modbus Stream",
+  "description": "Represents a modbus register.",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "RegisterAddress",
+      "schema": "integer",
+      "writable": true,
+      "displayName": "Register address",
+      "description": "Specifies the register address in the terminology of modbus RTU which gets used in order to read the register."
+    },
+    {
+      "@type": "Property",
+      "name": "FunctionCode",
+      "schema": "integer",
+      "writable": true,
+      "displayName": "Function Code",
+      "description": "Specifies the function code in the terminology of modbus RTU which gets used in order to read the register."
+    },
+    {
+      "@type": "Property",
+      "name": "DataType",
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+            {
+                "name": "UINT16",
+                "displayName": "UINT16",
```

Comment:
here we should always use Uint16 in order to be the same as all other enums

Reply:
Agree. Fixed.